doc.js

```diff
@@ -378,20 +378,27 @@ define('doc', ['event'], function(event) {
 				return !this.isPresent();
 			},
 
-			'on' : function(eventName, command, named) {
+			'on' : function(eventsName, command, named) {
+				var eventName = eventsName.split(' ');
 				this.each(function(el) {
-					event.addEvent(el, eventName, command, named);
-				});
-				event.boundEvents = event.boundEvents || {},
-					event.boundEvents[eventName] = event.boundEvents[eventName] || [],
-					boundElements = event.boundEvents[eventName];
-
-				this.each(function(el) {
-					if (boundElements.indexOf(el) === -1) {
-						boundElements.push(el);
+					for (var i = 0; i < eventName.length; i++) {
+						event.addEvent(el, eventName[i], command, named)
 					}
 				});
 
+				event.boundEvents = event.boundEvents || {};
+				for (var i = 0; i < eventName.length; i++) {
```

Comment:
Suggestion: you could put the old code, as it was, inside this loop. It would become simpler and more efficient.

```javascript
'on' : function(eventsName, command, named) {
  var events = eventsName.split(' ');
  for (var i = 0; i < events.length; i++) {
    var eventName = events[i];
    // old code here
  }
}
```

@luiz